testing/e2e/skr/skr-binding-test/index.js

```diff
@@ -0,0 +1,54 @@
+const {gatherOptions} = require('../skr-test');
+const {initializeK8sClient} = require('../utils/index.js');
+const {getSecret} = require('../utils');
+const {provisionSKRAndInitK8sConfig} = require('../skr-test/provision/provision-skr');
+const {deprovisionAndUnregisterSKR} = require('../skr-test/provision/deprovision-skr');
+const {KEBClient, KEBConfig} = require('../kyma-environment-broker');
+const keb = new KEBClient(KEBConfig.fromEnv());
+
+const provisioningTimeout = 1000 * 60 * 30; // 30m
+const deprovisioningTimeout = 1000 * 60 * 95; // 95m
+let globalTimeout = 1000 * 60 * 70; // 70m
+const slowTime = 5000;
+const secretName = 'sap-btp-manager';
+const ns = 'kyma-system';
+
+describe('SKR Binding test', function() {
+  globalTimeout += provisioningTimeout + deprovisioningTimeout;
+
+  this.timeout(globalTimeout);
+  this.slow(slowTime);
+
+  let options = gatherOptions(); // with default values
+  let skr;
+  let kubeconfigFromBinding;
+
+  before('Ensure SKR is provisioned', async function() {
+    this.timeout(provisioningTimeout);
+    skr = await provisionSKRAndInitK8sConfig(options, provisioningTimeout);
+    options = skr.options;
+  });
+
+  it('Create SKR binding', async function() {
+    try {
+      kubeconfigFromBinding = await keb.createBinding(options.instanceID);
+    } catch (err) {
+      console.log(err);
+    }
+  });
+
+  it('Initiate K8s client with kubeconfig from binding', async function() {
+    await initializeK8sClient({kubeconfig: kubeconfigFromBinding.credentials});
```

Comment:
Could you please add an assertion that where you cannot access cluster before initialization is done? I am afraid that it might swallow errors.

Reply:
I will change it to not provision and init in the first step.